pythainlp/translate/core.py

```diff
@@ -40,12 +40,10 @@ def download_model_all() -> None:
 
 
 def _get_translate_path(
-    model: str, path1: str, path2: str, file: str = None
+    model: str, *path1: str
```

Comment:
because we now have only one path variable, how about we just use `path` if it isn't a reserved word.